AntistasiOfficial.Altis/Missions/SUP_CitySupply.sqf

```diff
@@ -0,0 +1,246 @@
+if (!isServer and hasInterface) exitWith {};
+
+params ["_marker", "_type"];
+//[3,[],[],[],[],[]] params ["_countBuildings","_targetBuildings","_allGroups","_allSoldiers","_allVehicles","_leafletCrates"];
+private ["_targetPosition","_targetName","_duration","_endTime","_task","_spawnPosition","_missionVehicle","_crate","_range","_allBuildings","_usableBuildings","_index","_perimeterBuildings","_currentBuilding","_lastBuilding","_bPositions","_groupType","_params","_group","_dog","_leaflets","_drop"];
+
+
+_tskTitle = "City Supplies";
+_tskDesc = "City Supplies Description";
+_tskDesc_fail = "You failed delivering a paket, very impressive";
+_tskDesc_success = "You crazy little postman";
+
+_targetPosition = getMarkerPos _marker;
+_targetName = [_marker] call AS_fnc_localizar;
+_range = [_marker] call sizeMarker;
+
+_allBuildings = nearestObjects [_targetPosition, ["Building"], _range];
+if (count _allBuildings == 0) then {
+	while {(_range < 1000) AND (count _allBuildings == 0)} do {
+		_range = _range + 100;
+		_allBuildings = nearestObjects [_targetPosition, ["Building"], _range];
+	};
+};
+
+if (count _allBuildings == 0) exitWith {};
+
+//@Stef using a random buidling or perhaps a special building?
+_targetBuilding = selectRandom _allBuildings;
```

Comment:
@StefArma here is another question for you to answer. I wasn´t sure about it

Reply:
Land_i_Shed_Ind_F

and a distance check of 300m from HQ possibly

Reply:
As dropoff point in the city or as the point where you get this crate.

Creating a shed in the city will be easy, randomly create it somewhere is not really realistic, isn´t it?
The dropoff point will be guarded by two patrols, are more required?

Reply:
My bad:
The building that i indicated should be the building where the supplies can spawn inside, so you go in, kill enemies, get the supplies and bring them to town.

the location where to deliver can be simply the center of the town, ideally, wherever you drop the box within the town marker, you should be able to trigger the small qrf and hold the area for a minute that is the time civilians get close to get the supplies.